Penteract/Assets/Scripts/RangedAI.cpp

```diff
@@ -275,7 +295,10 @@ void RangedAI::OnCollision(GameObject& collidedWith, float3 collisionNormal, flo
 			if (collidedWith.name == "EMP") {
 				if (agent) agent->RemoveAgentFromCrowd();
 				stunTimeRemaining = stunDuration;
-				if (state != AIState::STUNNED) ChangeState(AIState::STUNNED);
+				if (state != AIState::STUNNED) {
+					ChangeState(AIState::STUNNED);
+					particlesEmp->PlayChildParticles();

```

Comment:
Check nullity. In the start you check the nullity of the aux local variable, but at this point this pointer could be null